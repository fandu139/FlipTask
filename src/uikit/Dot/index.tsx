import React from 'react';
import Colors from '../../theme/colors';
import { StyleSheet, View } from 'react-native';

const Dot: React.FC = () => <View style={styles.dot} />;

const styles = StyleSheet.create({
  dot: {
    backgroundColor: Colors.GRAY90,
    width: 5,
    height: 5,
    borderRadius: 5 / 2,
    marginVertical: 2,
  },
});

export default Dot;
