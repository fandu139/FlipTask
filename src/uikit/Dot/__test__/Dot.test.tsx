import Dot from '../index';
import React from 'react';
import { render } from '@testing-library/react-native';

describe('Dot component test', () => {
  describe('Dot snapshot test', () => {
    it('should render correctly', () => {
      const tree = render(<Dot />).toJSON();
      expect(tree).toMatchSnapshot();
    });
  });
});
