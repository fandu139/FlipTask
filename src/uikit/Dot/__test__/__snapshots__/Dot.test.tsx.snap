// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Dot component test Dot snapshot test should render correctly 1`] = `
<View
  style={
    Object {
      "backgroundColor": "#909090",
      "borderRadius": 2.5,
      "height": 5,
      "marginVertical": 2,
      "width": 5,
    }
  }
/>
`;
